### ISSUE TITLE
support document policies

### ISSUE THREAD

**Issue by BillGoldstein** [mikewest/origin-policy#64]
Support objects equivalent to the Document-Policy and Require-Document-Policy headers, as per
https://github.com/w3c/webappsec-feature-policy/blob/master/document-policy-explainer.md
which splits out several features currently defined in Feature-Policy header.
Possibly under the "feature" object?

**Comment by domenic** [comment]
Yeah, that's the plan, per https://github.com/WICG/origin-policy#configurable-policy-items. I guess we should add it to https://github.com/WICG/origin-policy/blob/master/policy-format.md#still-in-flux.

Work here (as well as on report-only feature policy) is mostly blocked on the dependent specs coming into existence.
